### PR TITLE
Ch feature type component

### PR DESCRIPTION
**Pull request by kelsiebesingeryeh**
- [ X] Feature
- [ ] Refactor
- [X ] Bug Fix 
- [ X] Clean Up

We split up the Category component into three different components (Eco, Vegan and AllergenFriendly) in order to display the product types for each category. We decided to split it up in order to ensure the data would display correctly. We also went ahead and styled each page with conditional rendering and using images to showcase the product types. We also cleaned up the class names and changed them to camelCase to be consistent. 

We encountered a major problem with the Category Component when trying to make it dynamic. We ended up splitting up the Category component into three smaller components and that way our Route links stayed the same and nothing ended up broken.

Reviewer should start in the Components folder, and go into the Category folder and review all of the components in the Category folder. Reviewer should also go into the Type component to review the Type component.

NPM Start and run localhost:3000 in the browser.

#### Checklist:
- [X ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [X ] My code follows the style guidelines of this project
- [ ] I have used tests to prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my change

### CODE REVIEW

File: src/components/App/App.css [hunk 1]
```diff
@@ -59,26 +59,26 @@ h1 {
...
+  .categoryContainer {
     flex-direction: row;
     height: 70vh;
   }
   .mainCategory {
     width: 24vw;
     height: 40vh;
   }
-  .mission-statement {
+  .missionStatement {
```
Comment (conconartist) [inline src/components/App/App.css:81]: Thanks for going through and fixing these!

File: src/components/App/App.css [hunk 2]
```diff
@@ -16,7 +16,7 @@ h2 {
   width: 80vw;
 }
 
-.category-container {
+.categoryContainer {
```
Comment (gaj23) [inline src/components/App/App.css:19]: Nice catch on Kelsie's part about maintaining naming conventions

File: src/components/App/App.js [hunk 3]
```diff
@@ -12,80 +14,91 @@ class App extends Component {
...
     const eco = response.filter(
       (item) =>
-        item["tag_list"].includes("Ewg verified")
-        // item["tag_list"].includes("Purpicks") ||
-        // item["tag_list"].includes("Certclean") ||
-        // item["tag_list"].includes("Ecocert") ||
-        // item["tag_list"].includes("Usda organic") ||
-        // item["tag_list"].includes("Non-gmo") ||
-        // item["tag_list"].includes("Organic")
+        item["tag_list"].includes("EWG Verified") ||
```
Comment (gaj23) [inline src/components/App/App.js:39]: Maybe these should be refactored into helper functions?

File: src/components/Category/AllergenFriendly.js [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+import Type from '../Type/Type';
+import './Category.css';
+
+const AllergenFriendly = ({ allergenFriendly }) => {
+    const filterMakeupTypes = allergenFriendly.map((item) => item["product_type"]);
+    const productTypes = filterMakeupTypes.filter(
+      (item, index) => filterMakeupTypes.indexOf(item) === index
+    );
+
+    const assignUrl = (item) => {
```
Comment (gaj23) [inline src/components/Category/AllergenFriendly.js:12]: Thank for you driving during this time Connie! I'm glad it worked out and we were able to apply standard styling to everything so far.

File: src/components/Category/Category.js [hunk 5]
```diff
@@ -1,11 +1,37 @@
...
 
-function Category() {
-    return (
-      <>
-        <h2>Does this work</h2>
-      </>
-    );
-}
+const Category = ({ data }) => {
+  const filteredMakeupTypes = data.map(item => {
```
Comment (gaj23) [inline src/components/Category/Category.js:5]: If we're unable to refactor all our categories into one come Monday, I think we should delete this file and make mention of it as an `wontfix` issue for the MVP of the project.

**Review by gaj23 — COMMENTED** [review]
I'm proud of the progress we made today! We hand a major tangle and we're able to problem solve together to smooth things out.

**Pull request merged**